stdlib/contrib/anaisdg/anomalydetection/mad.flux

```diff
@@ -1,9 +1,32 @@
+// Package anomalydetection detects anomalies in time series data.
 package anomalydetection
 
 
 import "math"
 import "experimental"
 
+// mad uses the median absolute deviation (MAD) algorithm to detect anomalies in a data set.
+//
+// Input data requires `_time` and `_value` columns.
+// Output data is grouped by `_time` and includes the following columns of interest:
+//
+// - **\_value**: difference between of the original `_value` from the computed MAD
+//   divided by the median difference.
+// - **MAD**: median absolute deviation of the group.
+// - **level**: anomaly indicator set to either `anomaly` or `normal`.
+//
+// ## Parameters
+// - threshold: Deviation threshold for anomalies.
```

Comment:
There is a fix that lets up add input data parameters `tables`/`table` to the docs now. When you rebase, this should pass lint now.

```suggestion
// - threshold: Deviation threshold for anomalies.
// - table: Input data. Default is piped-forward data (`<-`).
```